posthog/api/integration.py

```diff
@@ -110,6 +111,23 @@ def create(self, validated_data: Any) -> Any:
             instance = twilio.integration_from_keys()
             return instance
 
+        elif validated_data["kind"] == "databricks":
+            config = validated_data.get("config", {})
+            server_hostname = config.get("server_hostname")
+            client_id = config.get("client_id")
+            client_secret = config.get("client_secret")
+            if not (server_hostname and client_id and client_secret):
+                raise ValidationError("Server hostname, client ID, and client secret must be provided")
```

Comment:
Very likely not an issue, but just in case: This also raises a validation error if any of the values is provided as `""`, which is not quite the same as not being provided.

Again, likely not an issue as all those values seem like they should be something more useful than `""`, but just mentioning it.